manifests/init.pp

```diff
@@ -13,6 +13,8 @@
   String $conf_d,
   String $confdir,
   String $config_file,
+  Array $control_interface,
```

Comment:
This one doesn't look to get used, though, I see `$control_port` in the template.  A mixup perhaps?